### ISSUE TITLE
Table headers on the organize page are not applying role information in semantically meaningful ways

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1633]
### Description

Table headers on the Organize page currently have role=”button”: https://github.com/pressbooks/pressbooks/blob/66fda01dc395d98bddeecf9705fb520837530d09/templates/admin/organize.php#L129-L131 

These header cells are read as blank or skipped by the screen reader (depending on the browser), likely because the button role overrides the semantic information of the table header. Because <th> elements are not keyboard accessible by default, these buttons also do not currently receive keyboard focus.

We should consider adding the button role to an element, such as a span, nested within the table header, rather than the <th> element itself. We should also add a tabindex attribute to this internal element. For example: 
`<th><span role="button" tabindex="0">Show in Web</span></th>`

In addition, neither the purpose (check/uncheck all pages in column) nor the status (checked/unchecked) of these buttons is clear. We should add additional information through the use of an ARIA label to clarify what pressing the button does and the current state of the button (for screen reader users). 

For example, we should consider adding something like `aria-label="check/uncheck Show in Web for all Front Matter pages"` as well as an `aria-pressed` attribute. For example, if some buttons within the table are checked while others are not, the attribute should be `aria-pressed="undefined"`; if all boxes have been checked, it should be set to `aria-pressed="true"`, and if unchecked, `aria-pressed="false"`. Ideally the label attribute should change dynamically depending on the state of the buttons within the table.

Alternatively, we may with to place an actual checkbox in the header itself to allow users to explicitly select/deselect all items in that section column.

**Comment by mcgratay** [comment]
ARIA label is incorrect for follow elements: 
Show in Exports for Front Matter - Should be "check/uncheck Show in Exports for all Front Matter pages"
Show Title for Front Matter - Should be "check/uncheck Show Title for all Front Matter Pages"
Show in Exports for Back Matter - Should be "check/uncheck Show in Exports for all Back Matter pages" 
Show Title for Back Matter - Should be "check/uncheck Show Title for all Back Matter pages" 

Also -- for all Parts, ARIA label should be modified to "check/uncheck _________ for all Chapter pages **in this Part**" or it would otherwise only be true if the user had only one part in the book.

**Comment by mcgratay** [comment]
Resolved 👍
